crawl-ref/source/webserver/config.py

```diff
@@ -72,53 +85,166 @@
         ttyrec_path = "./rcs/ttyrecs/%n",
         socket_path = "./rcs",
         client_path = "./webserver/game_data/",
+        # dir_path = ".",
+        # cwd = ".",
         morgue_url = None,
-        send_json_options = True)),
-    ("seeded-web-trunk", dict(
-        name = "DCSS trunk, custom seed",
-        crawl_binary = "./crawl",
-        rcfile_path = "./rcs/",
-        macro_path = "./rcs/",
-        morgue_path = "./rcs/%n",
-        inprogress_path = "./rcs/running",
-        ttyrec_path = "./rcs/ttyrecs/%n",
-        socket_path = "./rcs",
-        client_path = "./webserver/game_data/",
-        morgue_url = None,
-        send_json_options = True,
-        options = ["-seed"])),
-    ("sprint-web-trunk", dict(
-        name = "Sprint trunk",
-        crawl_binary = "./crawl",
-        rcfile_path = "./rcs/",
-        macro_path = "./rcs/",
-        morgue_path = "./rcs/%n",
-        inprogress_path = "./rcs/running",
-        ttyrec_path = "./rcs/ttyrecs/%n",
-        socket_path = "./rcs",
-        client_path = "./webserver/game_data/",
-        morgue_url = None,
-        send_json_options = True,
-        options = ["-sprint"])),
-    ("tut-web-trunk", dict(
-        name = "Tutorial trunk",
-        crawl_binary = "./crawl",
-        rcfile_path = "./rcs/",
-        macro_path = "./rcs/",
-        morgue_path = "./rcs/%n",
-        inprogress_path = "./rcs/running",
-        ttyrec_path = "./rcs/ttyrecs/%n",
-        socket_path = "./rcs",
-        client_path = "./webserver/game_data/",
-        morgue_url = None,
+        # milestone_path = "./rcs/milestones",
         send_json_options = True,
-        options = ["-tutorial"])),
+        # env = {"LANG": "en_US.UTF8"},
+        )),
 ])
 
+def load_games():
```

Comment:
because of the complexity of dgamelaunch setups and how the handle config.py, and for the sake of easing transition, my instinct is to not put code like this in a module named `config.py`. One idea might be to introduce a new module entirely, e.g. `server_config.py` for servers that have moved to the new setup?

Reply:
That makes sense. I've split to a new file.